avocado/utils/memory.py

```diff
@@ -320,3 +320,56 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+class MemInfo(object):
+    """
+    Representation of /proc/meminfo
+    """
+
+    class MemInfoItem(object):
```

Comment:
I don't want to make you lose a lot of time refactoring this into a new library for data sizes, so for now I'd just take it out of `MemInfo` and renaming it as a `DataSize` class.